### ISSUE TITLE
Allow editing quality/speed in the editor

### ISSUE THREAD

**Issue by karaggeorge** [wulkano/Kap#903]
Right now, our conversion processes use good defaults for quality. For most conversions there's two parameters:

- `crf` which has lower and upper limits depending on the encoder, that sets the average bitrate, therefore how lossless the quality is (0 is lossless, but documentation says that values up to 17-18 appear pretty much lossless to the human eye)

- `preset` which can have values like `fast`, `slow` etc, showing how much time the encoder will spend. Making it faster, makes the conversion faster sacrificing quality and size, making it slower, means that it'll take longer but it'll get smaller size and better quality.

I'm thinking we can add a small button with an icon, that pops up a popup with 2 sliders (like fps). One for each of those, labeled `Quality` and `Speed` or something like that, defaulting to our current default values.

For gif, since we're using gifsicle I think we could use lossy instead.

Not sure if that's something people want/need so we can get some feedback. Maybe our good defaults are not worth adding more options. I've just seen a bunch of software allow at least one export `Quality` slider, allowing adjustments to quality/size.

**Comment by sindresorhus** [comment]
A quality slider might make sense, but I think a speed thing is too much. Also important to note that nobody has actually requested a quality setting, so not sure how much demand there might be for it, especially now that we have lossy compression for GIF.

**Comment by karaggeorge** [comment]
Yeah I agree. No one has asked for it, other than noticing gif sizes being too big, but with gifsicle that might be covered. I've just been messing with ffmpeg options a lot lately and saw it as a possible feature if we get requests for it at some point

**Comment by erawhctim** [comment]
Not sure if this is related/relevant here, but another idea I had when trying to slim down high-res videos: What about a resolution % button with some reasonable defaults (25/50/75/100)? When exporting captures of my android device, it'd be nice to be able to set the resolution to a smaller percentage (but keep the aspect ratio). I feel like I'd use a 50% or 75% option all the time.

**Comment by karaggeorge** [comment]
Do you mean something like #895 ? If so, that is currently in the works 😄

**Comment by erawhctim** [comment]
Yep! That's exactly what I was looking for :) thanks for the link, looking forward to it

**Comment by superdump** [comment]
Just noting that personally I would appreciate having the quality slider too. I know most people will prefer fast over better quality, but I would prefer the opposite. I'm generally using kap to screen capture stuff that I want to share with others. I'll capture once and it will be viewed many times. Quality is more important to me.

**Comment by alexmtrmd** [comment]
I definitely would like to see speed editor. Sometimes the recording made is too slow and i would like to speed it up.

**Comment by skllcrn** [comment]
Have you tried the `[kap-playback-speed](https://github.com/karaggeorge/kap-playback-speed)` plugin @alexmtrmd?

**Comment by alexmtrmd** [comment]
> Have you tried the [`kap-playback-speed`](https://github.com/karaggeorge/kap-playback-speed) plugin @alexmtrmd? 

No, but thank you.  Let me check it out

**Comment by devgonz** [comment]
I immediately noticed the poor quality in my screen capture. I needed this slider feature. I think the people who would request it are just abandoning for a different tool.

**Comment by timothyis** [comment]
> I immediately noticed the poor quality in my screen capture. I needed this slider feature. I think the people who would request it are just abandoning for a different tool.

Please feel free to contribute @devgonz 👍🏼 Much appreciated
